Production/python/customiseHLT.py

```diff
@@ -315,15 +317,32 @@ def customise(process):
                                      process.L1TableTask,
                                      process.caloTableTask,
                                      process.pixelTrackTableTask)
-  process.nanoSequenceMC = cms.Sequence(process.nanoTableTaskFS)
-  process.finalGenParticles.src = cms.InputTag("genParticles")
+  process.nanoTableTaskFS_data = cms.Task(
+                                     process.tauTablesTask,
+                                     process.tauTablesExtTask,
+                                     process.pfCandTablesTask,
+                                     process.AK4PFJetsTableTask,
+                                     process.recoAllGenJetsNoNuTask,
+                                     process.L1TableTask,
+                                     process.caloTableTask,
+                                     process.pixelTrackTableTask)
+
+  is_data =  True
```

Comment:
Should this assignment be removed? Otherwise the function will always run in `is_data = True` mode.